doc/user/data/sql_funcs.yml

```diff
@@ -324,6 +324,20 @@
   - signature: 'octet_length(b: bytea) -> int'
     description: Number of bytes in `b`
 
+  - signature: 'parse_ident(ident: str) -> str[]'
+    description: Splits qualified_identifier into an array of identifiers, removing any quoting of
+    individual identifiers. Extra characters after the last identifier are considered an error. Note
+    that this function does not truncate over-length identifiers. (From
+    [PG](https://www.postgresql.org/docs/current/functions-string.html))
+
+  - signature: 'parse_ident(ident: str, strict_mode: bool) -> str[]'
```

Comment:
Could we have it listed as `parse_ident(ident: str[, strict_mode: boo]l) -> str[]`? Only need one description then.

Reply:
Ah, I hadn't realized we moved to that style (among many others :) )